Std/Tactic/Omega/Logic.lean

```diff
@@ -5,15 +5,14 @@ Authors: Scott Morrison
 -/
 
 import Std.Tactic.Alias
-import Std.Logic
 
 /-!
 # Specializations of basic logic lemmas
 
 These are useful for `omega` while constructing proofs, but not considered generally useful
 so are hidden in the `Std.Tactic.Omega` namespace.
 
-If you find yourself needing them elsewhere, please move them first to `Std.Logic`.
+If you find yourself needing them elsewhere, please move them first to another file.
```

Comment:
I think the right message here is that `Std.Omega.Logic` should not be an import outside the implementation of `omega`. Users can still import `Std.Omega` as a whole and these lemmas come for free. It's unreasonable to expect users to inspect every lemma the use to see whether it comes from this file.

Reply:
Agreed.  We are working on getting omega into lean itself -- which provides a path to addressing this issue by making the lemmas part of the Lean package.  However, that update is not feasible in this PR.

Reply:
I can live with the current wording since it's not expected to last long.